experiments/snippets/dragAndDropOpportunityWall/README.md

```diff
@@ -0,0 +1,18 @@
+# Opportunity Planning Wall
```

Comment:
Why don't we call this what it is, a kanban wall? People love kanbans! This title really buries the lede.

Reply:
Maybe `Drag and Drop (Kanban) Opportunity Wall`